todo_app/data/session_items.py

```diff
@@ -55,6 +73,8 @@ def get_items():
                 status = ItemStatus.TODO
             elif idList == session[ItemStatus.FINISHED.name]:
                 status = ItemStatus.FINISHED
+            elif idList == session[ItemStatus.DOING.name]:
+                status = ItemStatus.DOING
             else:
                 status = -1
```

Comment:
Setting an enum isn't quite identical to setting a number, so if this line is hit then other parts of your code error! If you want a default status then I'd suggest setting here to TODO, but alternatively you could error and just insist that the names match (or even let the names be determined by environment variables)